src/cpp/r/session/graphics/RGraphicsDevDesc.hpp

```diff
@@ -55,6 +55,7 @@ struct DevDescVersion9
    Rboolean canGenMouseMove;
    Rboolean canGenMouseUp;
    Rboolean canGenKeybd;
+   Rboolean canGenIdle; // version 12
```

Comment:
I don't see you copying or initializing `canGenIdle` anywhere (e.g. similar to https://github.com/rstudio/rstudio/blob/fef100e920af3d51344ff8e4ae10556580f3e297/src/cpp/r/session/graphics/RShadowPngGraphicsHandler.cpp#L282).